### PR TITLE
Made numba as mandatory in README.md

### PR DESCRIPTION
**Pull request by AlphaVictorDelta**
Solved Issue #1146. 

This is my first open-source contribution ever.

### CODE REVIEW

**Comment by Yash-10** [comment]
Hello @AlphaVictorDelta thank you for this pull request and congratulations for your first open-source contribution.

I was thinking of this as a rather simple change by just removing "optional" for numba instead of writing "mandatory" since all the requirements are infact mandatory. I have also changed the issue title to better address it.

Also there are some CI failures (similar as in #1132) which we will have to deal with separately.

If you have any questions, please feel free to ask!

**Comment by AlphaVictorDelta** [comment]
@Yash-10 so should I delete this pull request and make another pull request?

**Comment by Yash-10** [comment]
@AlphaVictorDelta No need to delete this pull request. You can make the changes in this PR itself :)

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
     <a href="http://poliastro.github.io/"><image src="http://poliastro.github.io/images/logo_text.png" alt="poliastro logo" width="675px"></a>
 </p>
 
-Name:         | poliastro
-:------------:|:--------------
-**Website**:  | <https://poliastro.github.io/>
-**Author**:   | Juan Luis Cano Rodríguez [![orcid](https://img.shields.io/badge/id-0000--0002--2187--161X-a6ce39.svg)](http://orcid.org/0000-0002-2187-161X)
-**Version**:  | 0.15.dev0
+|    Name:     | poliastro                                                                                                                                    |
```
Comment (DhruvSondhi) [inline README.md:5]: Hello @AlphaVictorDelta ... I don't think this needs to be changed as it is already configured correctly ... Can you please revert this back?

File: README.md [hunk 2]
```diff
@@ -136,7 +138,7 @@ use of the library. Please refer to the COPYING file.
 poliastro comes from Polimi, which is the shortened name of the
 Politecnico di Milano, the Italian university where I was studying while
 writing this software. It\'s my tiny tribute to a place I came to love.
-*Grazie mille!*
+_Grazie mille!_
```
Comment (DhruvSondhi) [inline README.md:141]: Also, this can be reverted back ... GitHub works correctly with normal Markdown syntax :)

**Review by DhruvSondhi — CHANGES_REQUESTED** [review]
Please take a look at the requested changes :)

**Comment by AlphaVictorDelta** [comment]
@DhruvSondhi  Done!

**Comment by DhruvSondhi** [comment]
Hey @AlphaVictorDelta, I am not able to see the changes 🤔 ... Have you pushed the commits? & Please Squash all the commits as they are just solving a small issue :)

File: README.md (hunk 2 above)
Comment (DhruvSondhi) [reply README.md:141]: Same here !!

File: README.md (hunk 1 above)
Comment (DhruvSondhi) [reply README.md:5]: No changes can be seen 🤔

**Pull request closed without merging**